src/client/pydaos/SConscript

```diff
@@ -1,16 +1,18 @@
 """Build pydaos client"""
 import daos_build
+import sys
 
-def build_shim_module(env, lib_prefix, version):
+def build_shim_module(env, lib_prefix):
     """Build PyDAOS shim module for the specified python version"""
 
     if GetOption('help'):
         return
 
     new_env = env.Clone()
     extra_flags = ""
-
-    if version == "3":
+    version = "{}.{}".format(
+        sys.version_info.major, sys.version_info.minor)
```

Comment:
Optional: If you wanted to shorten this, `sys.version_info[0] == sys.version_info.major` and `sys.version_info[1] == sys.version_info.minor`, so you could use a python list slice:
```suggestion
    version = "{}.{}".format(*sys.version_info[0:2])
```